pandas/io/sas/sas_xport.py

```diff
@@ -482,7 +482,7 @@ def read(self, nrows=None):
 
         df = pd.DataFrame(index=range(read_lines))
         for j, x in enumerate(self.columns):
-            vec = data["s%d" % j]
+            vec = data[f"s{j}"]
```

Comment:
do you think this would be clearer as `"s" + str(j)`?  this strikes me as a borderline case